importer/importer/models.py

```diff
@@ -80,11 +81,12 @@ def check_total_item_count(self):
         actual_item_count = len(os.listdir(self.images_folder))
 
         if actual_item_count < int(self.item_count):
-            self.logger.error("Expected item count {0} but actual item count of {1} is {2}".format(
-                self.item_count,
-                self.images_folder,
-                actual_item_count
-            ))
+            self.logger.error("Expected item count %(item_count)d but actual item count of "
```

Comment:
I like this in part because it'll play better with the translation framework if we ever enable localization support